### PR TITLE
go: update & add 3rd p. packages  for gometalinter

### PR DESCRIPTION
**Pull request by mehmetalisavas**

### CODE REVIEW

**Review by cihangir — CHANGES_REQUESTED** [review]
please send glide.yaml update & lock file

File: go/src/glide.yaml [hunk 1]
```diff
@@ -223,6 +223,8 @@ import:
 - package: github.com/golang/snappy
 - package: github.com/hailocab/go-hostpool
 - package: gopkg.in/inf.v0
+- package: gopkg.in/alecthomas/kingpin.v2
+- package: gopkg.in/alecthomas/kingpin.v3-unstable
```
Comment (cihangir) [inline go/src/glide.yaml]: strange, gometalinter imports both https://godoc.org/github.com/alecthomas/gometalinter?imports

**Comment by mehmetalisavas** [comment]
PTAL, glide works a bit weird :/ but i've added glide.yaml & glide.lock fields..

**Comment by mehmetalisavas** [comment]
ref : https://github.com/koding/koding/pull/10477

**Review by cihangir — APPROVED** [review]

**Pull request merged**